### PR TITLE
Ignore media column if it is present

### PR DESCRIPTION
**Pull request by lognaturel**
Makes marginal progress towards https://github.com/XLSForm/pyxform/issues/601

#### Why is this the best possible solution? Were any other approaches considered?
It's not a great solution towards the original issue but I think it's an improvement. We don't have an example of header validation and I don't have time to design a good solution right now.

#### What are the regression risks?
A user could presumably be expecting an exception there now. That seems fine to break.

#### Does this change require updates to documentation? If so, please file an issue [here](https://github.com/XLSForm/xlsform.github.io) and include the link below.
No.

#### Before submitting this PR, please make sure you have:
- [x] included test cases for core behavior and edge cases in `tests`
- [x] run `nosetests` and verified all tests pass
- [x] run `black pyxform tests` to format code
- [x] verified that any code or assets from external sources are properly credited in comments

### CODE REVIEW

File: pyxform/survey.py [hunk 1]
```diff
@@ -774,7 +774,8 @@ def _set_up_media_translations(media_dict, translation_key):
             if parent and not parent.get("choice_filter"):
                 translation_key = survey_element.get_xpath() + ":label"
                 media_dict = survey_element.get("media")
-                _set_up_media_translations(media_dict, translation_key)
+                if type(media_dict) is dict:
```
Comment (lindsay-stevens) [inline pyxform/survey.py:777]: `if isinstance(media_dict, dict)`

File: pyxform/validators/pyxform/missing_translations_check.py [hunk 2]
```diff
@@ -88,7 +88,7 @@ def process_cell(typ, cell):
     if 0 < len(sheet_data):
         # e.g. ("name", "q1"), ("label", {"en": "Hello", "fr": "Bonjour"})
         for column_type, cell_content in sheet_data[0].items():
-            if column_type == constants.MEDIA:
+            if column_type == constants.MEDIA and type(cell_content) is dict:
```
Comment (lindsay-stevens) [inline pyxform/validators/pyxform/missing_translations_check.py:91]: `if isinstance(cell_content, dict)`

File: tests/test_sheet_columns.py [hunk 3]
```diff
@@ -75,6 +75,17 @@ def test_big_image_invalid_if_no_image(self):
...
 
+    def test_media_column__is_ignored(self):
+        self.assertPyxformXform(
+            name="data",
+            md="""
+            | survey |      |      |       |
+            |        | type | name | media |
+            |        | text | c    | m.png |
+            """,
+            xml__excludes=["m.png"],
```
Comment (lindsay-stevens) [inline tests/test_sheet_columns.py:86]: xml__xpath_match with a negative assertion, or xml__xpath_count with expected zero?

**Comment by lindsay-stevens** [comment]
> I don't have time to design a good solution right now

Is #601 a common or severe problem? The issue description and this PR don't seem bad to me, so some additional context might be useful for future reference.

**Review by lindsay-stevens — APPROVED** [review]

File: tests/test_sheet_columns.py (hunk 3 above)
Comment (lognaturel) [reply tests/test_sheet_columns.py:86]: Well... I don't even know what to not expect, exactly. 😅 I wanted to make sure that it wasn't defaulting to one of the media types.

**Comment by lognaturel** [comment]
> Is https://github.com/XLSForm/pyxform/issues/601 a common or severe problem?

I don't think so. How about we close #601 after this and I make a doc change: https://github.com/getodk/docs/issues/1546

**Pull request merged**